### PR TITLE
Purge stellate cache when KV updated

### PR DESCRIPTION
**Pull request by Razzmatazzz**
Purges specific API types when the corresponding KVs are updated.

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/data/changed_quests.json [hunk 1]
```diff
@@ -487,5 +487,47 @@
...
+                    "name": "Make ULTRA Great Again",
+                    "status": [
+                        "complete"
+                    ]
+                }
+            ]
+        }
+    },
+    "6086c852c945025d41566124": {
+        "name": "Revision - Reserve",
```
Comment (Shebuka) [inline src/tarkov-data-manager/data/changed_quests.json:520]: Also, **Revision - Lighthouse** and **Revision - Streets of Tarkov**

**Review by Shebuka — CHANGES_REQUESTED** [review]

File: src/tarkov-data-manager/data/changed_quests.json (hunk 1 above)
Comment (Shebuka) [reply src/tarkov-data-manager/data/changed_quests.json:520]: Actualy, from the wiki it seems like that the **Revision - Lighthouse** is given after you accept **Revision - Reserve**, and **Revision - Streets of Tarkov** requires both **Humanitarian Supplies** and **Friend from the West - Part 2**.

Comment (Razzmatazzz) [reply src/tarkov-data-manager/data/changed_quests.json:520]: Is Revision - Lighthouse available immediately after completing Humanitarian Supplies, or is it available once you start Revision - Reserve? The Wiki says the latter (which could be old info).

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/data/changed_quests.json (hunk 1 above)
Comment (Shebuka) [reply src/tarkov-data-manager/data/changed_quests.json:520]: I would trust wiki on this as it's new info added/updated after the wipe. I will complete **Humanitarian Supplies** today and report what happens to these two quests ;)

**Review by Shebuka — DISMISSED** [review]

File: src/tarkov-data-manager/data/changed_quests.json (hunk 1 above)
Comment (Shebuka) [reply src/tarkov-data-manager/data/changed_quests.json:520]: So, finishing **Humanitarian Supplies** has unlocked me **Revision - Reserve**.
As soon as I accepted **Revision - Reserve** both **Revision - Lighthouse** and **Revision - Streets of Tarkov** become available (I already completed _Friend from the West - Part 2_ so can't verify that that's a correct prerequisite, let's trust wiki :) )

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

File: src/tarkov-data-manager/data/changed_quests.json (hunk 1 above)
Comment (Razzmatazzz) [reply src/tarkov-data-manager/data/changed_quests.json:520]: Sounds good!

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by GrantBirki** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by GrantBirki — APPROVED** [review]

**Pull request merged**